### ISSUE TITLE
Hi, I added all the details to the python code and deploy to Heroku. The telegram bot is not responding. Any idea what's the issue?

### ISSUE THREAD

**Issue by melboo** [iamsushanth/telegram-chatbot-using-open-ai-python-heroku#1]

**Comment by prabhatkumar-debug** [comment]
did you change the openai and token of the bot

**Comment by melboo** [comment]
Yes, I changed the telegram token and the openai key but it's not working. Any idea? Also checked your demo at https://t.me/web3taskbot. It's not working either.

**Comment by prabhatkumar-debug** [comment]
can I know what type of issue you are facing or error
(because recently I have deployed this on Heroku and its working fine for me)

**Comment by prabhatkumar-debug** [comment]
https://t.me/Triganbossbot
this is a bot you can also check

**Comment by melboo** [comment]
Hi, thanks for your fast response. I am getting a H14 error, "no web processes running". Wonder if I need a change in the Procfile or dynos.

**Comment by geschenkwald** [comment]
same here, none of the bots is responding (the two demos and mine). do you have any suggestions why they aren't responding?

**Comment by iamsushanth** [comment]
Unfortunately, the bot has stopped after I launched it on the free Heroku server. Heroku's price has changed as a result of the fact that it is no longer free.

**Comment by rajaganapathy2000** [comment]
> can I know what type of issue you are facing or error (because recently I have deployed this on Heroku and its working fine for me)

root@localhost:~/CA# git push heroku heroku:main -f
error: src refspec heroku does not match any
error: failed to push some refs to 'https://git.heroku.com/th.git'
root@localhost:~/CA# git push heroku heroku:master -f

I'm getting this error while deploying
